### PR TITLE
fixing linear algebra and pandas

### PR DESCRIPTION
**Pull request by smolix**
*Description of changes:*

Fixing linear algebra and Pandas sections.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1848/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1848/

File: chapter_preliminaries/pandas.md [hunk 1]
```diff
@@ -110,16 +100,19 @@ X, y
 
 ## Summary
 
-* Like many other extension packages in the vast ecosystem of Python, `pandas` can work together with tensors.
-* Imputation and deletion can be used to handle missing data.
+We barely scratched the surface of what is possible, namely how to partition data columns and how to deal with missing variables in their most basic form. Later on, we will pick up more data processing skills in :ref:`sec_kaggle_house`. The relevant part for now is that we can easily convert tensors from Pandas into tensors managed by the framework of our choice.
```
Comment (AnirudhDagar) [inline chapter_preliminaries/pandas.md:103]: I think we can rephrase the last line a little.
`The relevant part, for now, is that we can easily convert data from Pandas into tensors managed by the framework of our choice.`

File: chapter_preliminaries/pandas.md [hunk 2]
```diff
@@ -132,3 +125,7 @@ Create a raw dataset with more rows and columns.
 :begin_tab:`tensorflow`
 [Discussions](https://discuss.d2l.ai/t/195)
 :end_tab:
+
+```{.python .input}
+
+```
```
Comment (AnirudhDagar) [inline chapter_preliminaries/pandas.md:131]: We can remove this empty tab.

**Review by AnirudhDagar — COMMENTED** [review]
@smolix although this has been merged, I've made a few small comments. PTAL. :))

**Pull request merged**